### ISSUE TITLE
Style CLI output

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#127]
@gustavocd put https://github.com/fatih/color on my radar in https://github.com/plentico/plenti/pull/125

We could make our CLI prints look more visually appealing by adding some color. I'd like to do this in some sort of standardized way if possible.

- https://github.com/spf13/cobra/issues/692#issuecomment-479631238
- https://medium.com/@inhereat/terminal-color-rendering-tool-library-support-8-16-colors-256-colors-by-golang-a68fb8deee86

**Comment by gustavocd** [comment]
I agree, I think we can start by improving the messages for the different commands and show some feedback like a spinner, then add some colors.

It'd be awesome to have a space to talk about it, maybe a slack channel or something 🤔, a place where we can share our findings.

**Comment by jimafisk** [comment]
Good point, better messages to provide users with some guidance is ultimately more important than aesthetics. You've made some great improvements already @gustavocd, so I'm definitely interested in hearing your thoughts. I'm down to chat in slack if we need a better way to share ideas, but for now feel free to brain dump thoughts and links here. If that becomes untenable we can always move the conversation. Thanks!

**Comment by jimafisk** [comment]
Emoji support: https://github.com/kyokomi/emoji

**Comment by jimafisk** [comment]
Color example from @zaydek: https://github.com/zaydek/retro/blob/master/pkg/term/term.go

**Comment by jimafisk** [comment]
https://github.com/charmbracelet/bubbletea

**Comment by jimafisk** [comment]
Progress bar option: https://github.com/schollz/progressbar

**Comment by ivanpirog** [comment]
**ColoredCobra**: https://github.com/ivanpirog/coloredcobra

**Comment by jimafisk** [comment]
Very cool @ivanpirog, thanks!
